interface/skin.dmf

```diff
@@ -144,69 +144,40 @@ window "mainwindow"
 		type = MAIN
```

Comment:
The 'Me' button at the bottom right is a bit odd looking at the moment, which I assume is just Dream Maker being Dream Maker and making phantom edits. Should be pretty easy to fix anyway.
![image](https://user-images.githubusercontent.com/57483089/128752097-3ce02471-f229-424b-be5d-526da6fe2283.png)
<hr>

Go to `skin.dmf` in Dream Maker and open the 'outputwindow' window, then select the 'Me' button in that, since it's the one having difficulties at the moment. Then open the edit window for it here:
![image](https://user-images.githubusercontent.com/57483089/128752563-865879dc-317e-4144-98be-701e3f1d2ca4.png)
Under the main 'General' tab in that, set the Border type to 'None', then go to the 'Config' tab and untick the 'Flat' option.
<hr>

That'll make a whole bunch of random edits in the `skin.dmf` file since apparently that's just how Dream Maker handles skin files, but it will fix the issue:
![image](https://user-images.githubusercontent.com/57483089/128753079-133081d0-7a9f-4908-8995-2edcdce7da72.png)

Reply:
Thanks, i also changed "Chat" to "Say"